app/src/main/java/com/github/onlynotesswent/ui/overview/editnote/Comments.kt

```diff
@@ -172,11 +174,11 @@ fun CommentsSection(
                       if (comment.isUnedited()) {
                         "${comment.userName} : $formattedDate"
                       } else {
-                        "${comment.userName} edited: $formattedDate"
+                        stringResource(R.string.edited, comment.userName, formattedDate)
```

Comment:
This is very well done, well played to figuring that string resources can take arguments